calculator/src/button.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import './button.css';
+const Button =() => {
```

Comment:
Try to give a Component name rather than keywords/ properties/ tag elements and give a meaningful name.